### PR TITLE
Add GraphCDN extension

### PR DESCRIPTION
**Pull request by mathieudutour**
## Description

List your [GraphCDN](https://graphcdn.io) services and inspect their configuration

## Type of change

- New extension

## Checklist

- [x] I read the [extension guidelines](https://developers.raycast.com/basics/prepare-an-extension-for-store)
- [x] I read the [documentation about publishing](https://developers.raycast.com/basics/publish-an-extension)
- [x] I ran `npm run build` and [tested this distribution build in Raycast](https://developers.raycast.com/basics/prepare-an-extension-for-store#metadata-and-configuration)

### CODE REVIEW

File: extensions/graphcdn/package.json [hunk 1]
```diff
@@ -0,0 +1,48 @@
...
+  "name": "graphcdn",
+  "title": "GraphCDN",
+  "description": "List your GraphCDN services and inspect their configuration",
+  "icon": "command-icon.png",
+  "author": "mathieudutour",
+  "license": "MIT",
+  "commands": [
+    {
+      "name": "search",
+      "title": "Search GraphCDN Services",
```
Comment (pernielsentikaer) [inline extensions/graphcdn/package.json:12]:
```suggestion
      "title": "Search Services",
      "subtitle": "GraphCDN",
```

File: extensions/graphcdn/package.json [hunk 2]
```diff
@@ -0,0 +1,48 @@
...
+      "description": "List your GraphCDN services and inspect their configuration",
+      "mode": "view"
+    }
+  ],
+  "preferences": [
+    {
+      "name": "token",
+      "type": "textfield",
```
Comment (pernielsentikaer) [inline extensions/graphcdn/package.json:20]: Could be a password field

File: extensions/graphcdn/src/search.tsx [hunk 3]
```diff
@@ -0,0 +1,270 @@
...
+  }, [setOrgs, setServices]);
+
+  const onFetchConfig = async (serviceName: string) => {
+    setLoading(true);
+    try {
+      const config = await getConfig(serviceName);
+      push(
+        <Detail
+          markdown={`\`\`\`\n${JSON.stringify(config, null, "  ")}\n\`\`\``}
+          actions={<ActionPanel></ActionPanel>}
```
Comment (pernielsentikaer) [inline extensions/graphcdn/src/search.tsx:68]:
```suggestion
          actions={<ActionPanel/>}
```
Could be collapsed or removed

**Review by pernielsentikaer — CHANGES_REQUESTED** [review]
Hi @mathieudutour 🚀 

Awesome extensions, I left a few minor suggestions for you to check 💪 

Request a new review when you are ready, feel free to contact me here or at [Slack](https://raycastcommunity.slack.com/) if you have any questions.

**Review by pernielsentikaer — CHANGES_REQUESTED** [review]
Hi @mathieudutour 🚀 

Much better 💪 !

Just a single question left:
Is it supposed to be able to create an organization, I can see some functions that's intended to do this, but not in use – Just wanted to be sure that nothing is missing 😄

**Comment by mathieudutour** [comment]
Not yet no, Just added the functions to do so if I (or someone else) want to add this functionality

**Review by pernielsentikaer — APPROVED** [review]
Awesome 💪  Good idea to keep it, just wanted to make sure we didn't lose anything 😄 

I can't find anything else that should be addressed!

**Pull request merged**